### PR TITLE
draft for CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by fMercury**
I think that many lines that now are in the README file must be in this file, but I will wait for reviews.

### CODE REVIEW

**Comment by kvbik** [comment]
hello @fMercury thx very much for the PR!

few comments>

1. fix the numbering in the document, seems like every paragraph should start with 1
2. in "Contributing with an Issue" there it starts as a numbered list and afterwards there is a bullet point - is it intended?
3. not sure about the `proposal/next` branch idea...
4. what is the inspiration for you version of the document? maybe when we see that one, we can understand the structure better

sorry for digging into this  that much, but i believe it is crucial to have this document right. that was one of the reasons why we postponed adding one. i don't think this file *must* exist. maybe not if necessary information were in the `README` file. but (a) we do not have them all there and (b) it is definitely very good practice to have `CONTRIBUTION` document. so - thx again for the work!!!

**Comment by kvbik** [comment]
tiny detail - not that i insist on rebasing your commits, but it is easier to accept a pull request without some merge commits and "removing blanks" commits.

**Comment by JirkaChadima** [comment]
@kvbik This looks very good and nicely generic. Maybe we could use it as is in our other repositories? It would close https://trello.com/b/ZREEkuOX/winding-tree-work-in-progress

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# How to contribute
+
+By participating in this project there are a few guidelines that we need contributors to follow:
+
+## Getting Started 
+ . Login with your Github account or create one now
```
Comment (kvbik) [inline CONTRIBUTING.md]: still a bit of a catch, use `*` instead. sorry for my OCD here ;)

Comment (kvbik) [reply CONTRIBUTING.md]: or `1.` or something like that.

**Comment by kvbik** [comment]
for sure! it is generic enough. i  will be  happy to merge (maybe update little bit) and spread around our repositories... or just put to our `wiki` repo and just link  it.

**Pull request merged**